src/components/Card.jsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import heroHeader from "../assets/hero-header.png";
+
+const Card = ({ item }) => {
+  return (
+    <div>
+      <div className="card e-commerce product-display-card">
+        <div className="container1 badge-card">
+          <img className="card-image" src={item.imageURL} />
+          <i className="image-badge-wishlist far fa-heart"></i>
+        </div>
+
+        <div className="container2">
+          <div className="card-description"> {item.description} </div>
+          <div className="price">
+            <div className="discount-price"> {item.price} </div>
+            <div className="original-price">
+              <s> {item.originalPrice} </s>
```

Comment:
Good use of semantic Element got to know that  this tag `<s>` is used for strickthrough